samples/client/petstore/typescript-jquery/npm/model/InlineObject1.ts

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
please delete this file

Reply:
@macjohnny I'm not sure where these come from, but the file is referenced in https://github.com/OpenAPITools/openapi-generator/pull/3969/files/c99fcd7b5936800b4ef0d38e92ebd27dc86fadcb#diff-969fc1a4f92fd269b7aac842f2049bb7R15 as well. This is just the output of the build step.